app/views/shipping/shipping.scala.html

```diff
@@ -0,0 +1,57 @@
+@(subscription: model.Subscriptions.SubscriptionProduct)
+
+@import utils.Prices._
+
+@main(s"${subscription.capitalizedTitle} | Choose a Package | Subscriptions | The Guardian", isInternational=true) {
+
+    <main class="page-container gs-container">
+
+        @fragments.page.header(subscription.title, Some(subscription.description), Nil)
+
+
+        <section class="section-slice">
+            <h2 class="page-heading">@subscription.stepsHeading</h2>
+            <ol class="steps">
+                @for(step <- subscription.steps) {
+                    <li class="steps__item">@step</li>
+                }
+            </ol>
+        </section>
+
+        <section class="section-slice">
+
+            @if(subscription.insideM25) {
```

Comment:
Yeah, less concerned about the repetition of markup (I think that's fine), just about what the if statement was. Not a big thing though.